plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1350,7 +1350,11 @@ public function disable_compat_mode(): void {
 	 */
 	private function toggle_compat_mode( bool $enabled ): void {
 		if ( ! $this->synchronizer->check_orders_table_exists() ) {
-			WP_CLI::error( __( 'HPOS tables do not exist.', 'woocommerce' ) );
+			if ( $enabled ) {
```

Comment:
This is a change from `trunk` that isn't merged into our feature branch yet. Will fix.